src/test/ui/mir/mir_codegen_calls_diverging_drops.rs

```diff
@@ -2,6 +2,7 @@
 // error-pattern:diverging_fn called
 // error-pattern:0 dropped
 // ignore-emscripten no processes
+// needs-unwind
```

Comment:
Add a comment here about why this test needs unwind. Suggestion: "this test checks that a destructor is called after panicking".